File: api/src/main/java/net/blueshell/api/repository/CommitteeRepository.java
package net.blueshell.api.repository;

import net.blueshell.api.base.BaseRepository;
import net.blueshell.api.model.Committee;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface CommitteeRepository extends BaseRepository<Committee> {
    List<Committee> findALlByMembersUserIdEquals(Long userId);
}

File: api/src/main/kotlin/net/blueshell/api/repository/committee/CommitteeRepository.kt
package net.blueshell.api.repository.committee

import net.blueshell.api.base.BaseRepository
import net.blueshell.api.model.committee.Committee
import org.springframework.stereotype.Repository

@Repository
interface CommitteeRepository : BaseRepository<Committee, Long> {
    fun findAllBy_membersUserIdEquals(userId: Long): MutableList<Committee>
}
